src/modelfree/shaping_wrappers.py

```diff
@@ -35,8 +35,10 @@ def __init__(self, venv, agent_idx, shaping_params, reward_annealer=None):
 
         self.reward_annealer = reward_annealer
         self.agent_idx = agent_idx
-
+        self.log_buffers = defaultdict(lambda: deque([], maxlen=10000))
```

Comment:
There's a lot of code duplication here between `log_buffers` and `ep_rew_dict`. There's got to be a better way. Can we use just one of them, and only look at relevant slices of it when logging / getting data for the annealer? Or still have two objects, but put them in a dict and have everything iterate over the dict with shared logic?